src/screens/staking-validators.tsx

```diff
@@ -1,26 +1,12 @@
-import React, {useCallback, useEffect, useRef, useState} from 'react';
+import React, {useCallback} from 'react';
```

Comment:
why you move logic which should be used only on this screen into hooks?

Reply:
in three places

![image](https://user-images.githubusercontent.com/6774813/204483594-48cacb11-ee5c-4b56-b9f0-c891ec446a22.png)